### DISCUSSION TITLE
Sonnenkraft  SK-HWR-8

### DISCUSSION THREAD

**Discussion by krema445** [nathanmarlor/foxess_modbus#458] [Support] [answered]
I have an SK-HWR-8 from Sonnenkraft which is identical to the FoxESS H3 8.0 E. This inverter from Foxess is sold by the company Sonnenkraft.
When integrating the inverter I get the following error message:
Error connecting to your adapter or inverter. Ensure the adapter is properly configured and is correctly wired to your inverter (see the setup link above), then try again. Details: **Inverter model not supported: 'SK-HWR-8**'
Is there a way to still integrate the inverter via RS485?

**Comment by canton7** [comment]
We'll need to update the integration - I'll try and get it done over the weekend

**Comment by canton7** [comment]
Would you mind trying out the pre-release version v1.8.0b2, see if that works for you?

**Comment by krema445** [comment, marked as answer]
v1.8.0b2 works perfectly - THANK YOU!!

![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/1fabc0ae-37c4-46c9-910f-5dbd0d8cb059)

**Reply by canton7** [reply]
Oops, ignore that last message, got my issues confused.

Glad it's working! I'll get that released properly soonish.

**Comment by krema445** [comment]
Hello.

 

The inverter in 3 phases. How can these values be presented in summary? In addition, the 2 strings of the solar modules are not displayed.

 

Thanks Martin

 

Von: Antony Male ***@***.***> 
Gesendet: Samstag, 4. November 2023 15:17
An: nathanmarlor/foxess_modbus ***@***.***>
Cc: krema445 ***@***.***>; Author ***@***.***>
Betreff: Re: [nathanmarlor/foxess_modbus] Sonnenkraft SK-HWR-8 (Discussion #458)

 

Oops, ignore that last message, got my issues confused.

Glad it's working! I'll get that released properly soonish.

**Reply by canton7** [reply]
What do you mean, "How can these values be presented in summary"?

Search for "PV1" and "PV2" in the list of entities to see power/voltage/current for the two strings.

**Comment by krema445** [comment]
Other entities are made available in the FoxESSCloud integration.

 

PV1 Power

PV2 Power
PV Power 

 

see link : https://github.com/macxq/foxess-ha#-provided-entities

 

Isn't this available in Modbus?

 

 

Von: Antony Male ***@***.***> 
Gesendet: Sonntag, 5. November 2023 11:31
An: nathanmarlor/foxess_modbus ***@***.***>
Cc: krema445 ***@***.***>; Author ***@***.***>
Betreff: Re: [nathanmarlor/foxess_modbus] Sonnenkraft SK-HWR-8 (Discussion #458)

 

What do you mean, "How can these values be presented in summary"?

Search for "PV1" and "PV2" in the list of entities to see power/voltage/current for the two strings.

**Reply by canton7** [reply]
I'm confused as to why you're having trouble. The entities "PV1 Power", "PV2 Power" and "PV Power" are provided by this integration and they literally have the same names. If you look at the list of entities provided by this integration you can't fail to see them.

**Comment by krema445** [comment]
These are mine - all PV values ​​are missing and also a few others.

 

Should I reinstall Homeassistant?

 







Von: Antony Male ***@***.***> 
Gesendet: Sonntag, 5. November 2023 14:06
An: nathanmarlor/foxess_modbus ***@***.***>
Cc: krema445 ***@***.***>; Author ***@***.***>
Betreff: Re: [nathanmarlor/foxess_modbus] Sonnenkraft SK-HWR-8 (Discussion #458)

 

I'm confused as to why you're having trouble. The entities "PV1 Power", "PV2 Power" and "PV Power" are provided by this integration and they literally have the same names. If you look at the list of entities provided by this integration you can't fail to see them.

**Reply by canton7** [reply]
Did you forget to include an image?

**Comment by krema445** [comment]
Sorry 😊

 

Von: Antony Male ***@***.***> 
Gesendet: Sonntag, 5. November 2023 14:19
An: nathanmarlor/foxess_modbus ***@***.***>
Cc: krema445 ***@***.***>; Author ***@***.***>
Betreff: Re: [nathanmarlor/foxess_modbus] Sonnenkraft SK-HWR-8 (Discussion #458)

 

Did you forget to include an image?

**Reply by canton7** [reply]
I think you need to use the github web interface to upload images, rather than replying to the emails.

**Comment by krema445** [comment]
![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/f56fbc20-29fe-46ff-9c18-6fc10d79e67d)
![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/15a51da1-8b1c-4e2a-848f-7a39032ed29f)
![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/596e9ffb-5d70-4f8e-ad07-874bd50f746a)

**Reply by canton7** [reply]
I'm not sure what page you're looking at there. Go to Settings - Devices & Services - FoxESS - Modbus - "1 device", and you'll see the list of entities.

**Comment by krema445** [comment]
![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/438b7795-fd5d-4903-974a-a4607286e948)
![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/cdddb34b-5f7f-48c3-828f-2954834cdf10)
![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/f1ece4c3-d7bd-4375-a130-89c8f8207886)
![image](https://github.com/nathanmarlor/foxess_modbus/assets/136677564/2127bb08-db5e-4776-ae6b-cd894a5dbd12)

**Reply by canton7** [reply]
D'oh, I never merged in #450, my bad! Try v1.8.0b4.

**Comment by krema445** [comment]
Hello - thank you again for your support.

So far everything is working without any glitches.
I'm still missing a few entities - which are available in the cloud version.
For example:
Grid Consumption Power
Feedin Power
etc.
But there's no rush - thanks in advance 
Martin

**Reply by canton7** [reply]
I can see those entities in your screenshots? That said, see #462
